### DISCUSSION TITLE
Getting error with PBValue

### DISCUSSION THREAD

**Discussion by mat-tgn** [tttapa/Control-Surface#1081] [Q&A]
Validating the code  from PB Example on board Arduino Leonardo i get this error 

"Compilation error: 'PBValue' does not name a type"

It must be something simple, but I can't figure out what I'm doing wrong.

` 
#include <Control_Surface.h>
 
USBMIDI_Interface midi;
 
// Instantiate an object that listens for MIDI Pitch Bend messages on channel 1.
PBValue pb {Channel_1};
 
void setup() {
  Control_Surface.begin();
  Serial.begin(115200);
}
 
void loop() {
  Control_Surface.loop();
  // If the Pitch Bend value changed
  if (pb.getDirty()) {
    // Print it
    Serial.println(pb.getValue());
    // Clear the dirty flag to acknowledge the change
    pb.clearDirty();
  }
}`

**Comment by tttapa** [comment]
Which version of Control Surface are you using?
